ek9000App/src/ekUtil.h

```diff
@@ -251,6 +256,11 @@ template <> inline bool setupCommonDpvt<aoRecord>(aoRecord* prec, TerminalDpvt_t
 	return setupCommonDpvt(prec->name, prec->out.value.instio.string, dpvt);
 }
 
+inline bool parseInt(const std::string& s, int& out) {
+	bool hex = s[0] == '0' && s[1] == 'x';
```

Comment:
Check for `s.length() > 2`?

Er, wait, where did this function sneak in? Doesn't seem used anywhere...

Reply:
Wow I wrote a useful function and didn't even use it...